include/swift/Basic/BlotMapVector.h

```diff
@@ -85,27 +88,34 @@ class BlotMapVector {
     }
 
     const_iterator find(const KeyT &Key) const {
-      return const_cast<BlotMapVector &>(*this)->find(Key);
+      return const_cast<BlotMapVector &>(*this).find(Key);
     }
 
-    /// This is similar to erase, but instead of removing the element from the
-    /// vector, it just zeros out the key in the vector. This leaves iterators
-    /// intact, but clients must be prepared for zeroed-out keys when iterating.
-    void erase(const KeyT &Key) { blot(Key); }
-
-    /// This is similar to erase, but instead of removing the element from the
-    /// vector, it just zeros out the key in the vector. This leaves iterators
+    /// Eliminate the element at `Key`. Instead of removing the element from the
+    /// vector, just zero out the key in the vector. This leaves iterators
     /// intact, but clients must be prepared for zeroed-out keys when iterating.
+    ///
+    /// Return true if the element was erased.
+    bool erase(const KeyT &Key) { return blot(Key); }
+
+    /// Eliminate the element at the given iterator. Instead of removing the
+    /// element from the vector, just zero out the key in the vector. This
+    /// leaves iterators intact, but clients must be prepared for zeroed-out
+    /// keys when iterating.
     void erase(iterator I) { erase((*I)->first); }
```

Comment:
Why doesn't this also return a bool?

Reply:
Presumably you're passing a valid iterator, so wouldn't make much sense.

Reply:
ok.